gradle/spec/fixtures/script_plugins/dependencies.gradle.kts

```diff
@@ -0,0 +1,39 @@
+ext {
+    libraries = listOf(:)
```

Comment:
Concern: this may also be a `mapOf()`

I expected this to read closer to: https://pl.kotl.in/5yG6h-nov